git-merge-pr

```diff
@@ -100,9 +100,10 @@ if [ $CURRENT_BRANCH = $BRANCH_TO_MERGE ] &&  [ $PRUNE_FLAG = off ]; then
 fi
 
 echo "Stashing any local changes and checking out remote branch 'origin/$DESTINATION_BRANCH'"
-STASH_OUTPUT="$(git stash save -a)" || exit 1;
+STASH_OUTPUT="$(git -c commit.gpgSign=false stash save -a)" || exit 1;
```

Comment:
what?!? heresy!

Reply:
is this bad? why would we need to gpgsign stash commits?

Reply:
update from Jeff: this is not bad

Reply:
Not bad